docs/e2eTestPrompts.md

```diff
@@ -360,6 +360,12 @@ This file contains prompts used for end-to-end testing to ensure each tool is in
 | azmcp_sql_server_firewall-rule_list | List all firewall rules for SQL server <server_name> |
 | azmcp_sql_server_firewall-rule_list | Show me the firewall rules for SQL server <server_name> |
 | azmcp_sql_server_firewall-rule_list | What firewall rules are configured for my SQL server <server_name>? |
+| azmcp_sql_server_firewall-rule_create | Create a firewall rule for my Azure SQL server <server_name> |
+| azmcp_sql_server_firewall-rule_create | Add a firewall rule to allow access from IP range <start_ip> to <end_ip> for SQL server <server_name> |
+| azmcp_sql_server_firewall-rule_create | Create a new firewall rule named <rule_name> for SQL server <server_name> |
```

Comment:
is "Update" supported by create?

Reply:
yep 'update' is P1 task after create/delete/list firewall rule P0 tasks. 
https://learn.microsoft.com/en-us/cli/azure/sql/server/firewall-rule?view=azure-cli-latest

I am trying to complete all P0 tasks for sql server, sql db CRUD by global hackathon week (it will provide base methods for other sql folks who plan to use MCP for their hackathon project). firewall rule create/delete/list is the mandatory config for sql server creation.